src/nunit-gui/Views/XmlView.cs

```diff
@@ -45,6 +45,9 @@ public XmlView()
             InitializeComponent();
 
             XmlPanel = new ControlElement<Panel>(xmlPanel);
+            selectAllToolStripMenuItem.Click += (s, a) => xmlTextBox.SelectAll();
+            copyToolStripMenuItem.Click += (s, a) => xmlTextBox.Copy();
+            wordWrapToolStripMenuItem.CheckedChanged += (s, a) => xmlTextBox.WordWrap = wordWrapToolStripMenuItem.Checked;
```

Comment:
I was a bit uncertain what to do here, since it is pure gui-stuff happening here. I don't mind moving it.